source/_integrations/modbus.markdown

```diff
@@ -557,6 +557,11 @@ sensors:
       required: false
       default: int
       type: string
+    data_size:
```

Comment:
as discussed in the core PR, data_size is not a good word suggestion was register_size.

Why because if you want to read 4 registers and define that in your structure, you would use count=4 and thus have a data_size of 8.

If you use register_size, and do e.g, count=4, register_size =4 you would read 4 registers and have a data_size of 16.